test/Operators/hybrid/2d.jl

```diff
@@ -150,6 +150,49 @@ end
     @test conv_adv_c2c[3] ≈ 2 atol = 0.1
 end
 
+@testset "1D SE, 1D FD Extruded Domain Discrete Product Rule Operations: Kinetic Energy" begin
+
+    gradc2f = Operators.GradientC2F(
+        top = Operators.SetValue(0.0),
+        bottom = Operators.SetValue(0.0),
+    )
+    gradf2c = Operators.GradientF2C()
+
+    n_elems_seq = 2 .^ (5, 6, 7, 8)
+    err, Δh = zeros(length(n_elems_seq)), zeros(length(n_elems_seq))
+
+    for (k, n) in enumerate(n_elems_seq)
+        # Discrete Prodouct Rule Test
+        # ∂(ab)/∂s = a̅∂b/∂s + b̅∂a∂s
+        # a, b are interface variables, and  ̅ represents interpolation
+        # c is a cell centered variable
+        # s is the coordinate along horizontal surfaces (terrain following)
+        # For this test, we use a(z) = z and b = sin(z),
+        hv_center_space, hv_face_space = hvspace_2D(helem = n, velem = n)
```

Comment:
can we test with a stretched mesh? a lot of issues only show up when the vertical isn't uniforrm

Reply:
added stretched test